clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -350,7 +370,10 @@ public AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs) {
      * before the replica has been created on the broker. It will support moving replicas that have already been created after
      * KIP-113 is fully implemented.
      *
-     * This operation is supported by brokers with version 1.0.0 or higher.
+     * Similar to #{@link AdminClient#alterReplicaDir(Map, AlterReplicaDirOptions)},
+     * but uses the default options. See {@link #alterReplicaDir(Map, AlterReplicaDirOptions)} for more details
+     *
+     * This operation is supported by brokers with version 1.0 or higher.
```

Comment:
Since we're saying 0.11.0.0 or higher elsewhere, it should be 1.0.0 or higher here, for consistency.

Reply:
@ijuma Do you know why `kafka.api.ApiVersion` uses `1.0` instead of `1.0.0`? The reason I use 1.0 here is because user have to specify `interBrokerProtocolVersion` as `1.0` rather than `1.0.0` in the future.

Reply:
@ijuma Never mind... I just realized that we say `0.11.0.0` in javadoc while the protocol in ApiVersion is `0.11.0`. I will update the java doc as you suggested.